HaulageApplication/HaulageTests/CustomerPageTests.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+using Haulage;
+using Haulage.BaseClasses.Accounting;
+using Moq;
+
+namespace HaulageTests
+{
+    public class CustomerPageTests
+    {
+        [Fact]
+        public void CustomerPage_ShouldInitializeWithoutParameters()
+        {
+            // Arrange & Act
+            var exception = Record.Exception(() => new CustomerPage());
+
+            // Assert
+            Assert.Null(exception);
+        }
+
+        [Fact]
+        public void CustomerPage_ShouldInitializeWithParameters()
+        {
+            // Arrange
+            var customer = new Customer();
+            var account = new Account(Guid.NewGuid(), "Billing Details", "Contact Details");
+
+            // Act
+            var exception = Record.Exception(() => new CustomerPage(customer, account));
+
+            // Assert
+            Assert.Null(exception);
+        }
+
+        //[Fact]
+        //public void SaveButton_Click_ShouldUpdateAccountDetails()
+        //{
+        //    // Arrange
+        //    var customer = new Mock<Customer>();
+        //    var account = new Account(Guid.NewGuid(), "Old Billing Details", "Old Contact Details");
+        //    var page = new CustomerPage(customer.Object, account);
+
+        //    // Simulate user input
+        //    page.FindByName<Entry>("BillingDetailsEntry").Text = "New Billing Details";
+        //    page.FindByName<Entry>("ContactDetailsEntry").Text = "New Contact Details";
+
+        //    // Act
+        //    page.SaveButton_Click(null, null);
```

Comment:
should this be null here and uncommented code either remove are try and get working

Reply:
This has been removed as this is now part of a future development task. Task #24.